packages/module/patternfly-docs/content/extensions/virtual-assistant/design-guidelines.md

```diff
@@ -0,0 +1,206 @@
+1. **Name:** Identifier for your ChatBot and the user. Choose a name for your ChatBot that users can easily identify as a bot. 
+1. **Label:** Labels ChatBot messages as "AI."
+1. **Timestamp:** The relative or absolute time that a message was sent.
+1. **Quick responses:** Programmable, clickable actions that allow users to quickly answer questions from the ChatBot.
+1. **Sources:** Cards that link to documentation or other external resources. When multiple sources are included, users can paginate through the different options. 
+1. **Response actions:** Actions that allow users to interact with a bot message. these typically include providing feedback, copying, sharing, or reading aloud, but [custom message actions](/patternfly-ai/chatbot/messages#custom-message-actions) are also supported.
+
+### Message bar 
+
+To message the ChatBot, users can type directly into the message bar in the footer or click any included actions. 
+
+<div class="ws-docs-content-img">
+![Elements of a message bar, including input actions.](./img/message-bar-elements.svg)
+</div>
+
+1. **Attach button:** Allows users to upload files from their computer. 
+1. **Use microphone button:** Supports speech recognition to allow users to use voice input. This feature is currently only available in Chrome and Safari.
+1. **Send button:** Allows users to send a typed message. This button should be disabled until a user has input text. 
+
+When a user chooses to use speech input via the microphone button, the button will display an animation to indicate that the ChatBot is listening to the user (as shown in [this speech recognition example](/patternfly-ai/chatbot/ui#message-bar-with-speech-recognition-and-file-attachment)). To stop voice input, users will need to select the microphone button again.
+
+<div class="ws-docs-content-img">
+![Active listening button in the message bar.](./img/listening.svg)
+</div>
+
+When a user selects the "listen" [message action](/patternfly-ai/chatbot/messages#messages-actions) on a bot message, the message contents will be read aloud and a stop button will be displayed as the only action in the message bar. Selecting this button will halt the message playback.
+
+<div class="ws-docs-content-img">
+![Stop button in the message bar.](./img/stop-button.svg)
+</div>
+
+### Footnotes 
+
+The footnote provides a persistent space to display messaging about your product's Terms and Conditions, which focus on the rules of using the service, and Privacy Policy, which focuses on the handling of personal data.
+
+Though footnotes are not required, they are highly recommended to ensure legal compliance, establish user trust, and clearly define the usage guidelines and data handling practices. 
+
+When users select the footnote, you can display a popover that provides more information than would fit in the footnote:
+
+<div class="ws-docs-content-img">
```

Comment:
<img width="544" alt="Screenshot 2024-11-25 at 7 06 00 PM" src="https://github.com/user-attachments/assets/90c91f8e-7aab-48e2-86fd-e65723d81b8f">

It looks like this in reality ^.

Reply:
The screenshot in the design guidelines has the footer popup a little higher than it is in reality, but it's not a blocker from my side.